<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainEditorForm</class>
 <widget class="QDialog" name="mainEditorForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>911</width>
    <height>791</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>mainEditorForm</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Select Transport</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Select Distance</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>190</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Select Average Speed</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>260</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>What will you compare?</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbTransport">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>79</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(122, 122, 122);</string>
   </property>
   <item>
    <property name="text">
     <string>Car</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Bus</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Train</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Walking</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="cmbDistance">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>79</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(122, 122, 122);</string>
   </property>
   <item>
    <property name="text">
     <string>5km</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10km</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>60km</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>100km</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="cmbSpeed">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>210</y>
     <width>79</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(122, 122, 122);</string>
   </property>
   <item>
    <property name="text">
     <string>5km/hr</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>60km/hr</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>120km/hr</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>200km/hr</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="lblShow">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>380</y>
     <width>721</width>
     <height>181</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(255, 255, 255);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>Hello</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="chbPrice">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>280</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(122, 122, 122);
border-radius: 8px;
padding: 4px;
</string>
   </property>
   <property name="text">
    <string>Price</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chbEnergy">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>310</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(122, 122, 122);
border-radius: 8px;
padding: 4px;</string>
   </property>
   <property name="text">
    <string>Energy Consumption per km</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chbConversions">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>340</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(122, 122, 122);
border-radius: 8px;
padding: 4px;</string>
   </property>
   <property name="text">
    <string>Energy Conversions</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnGen">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>130</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 8px;
background-color: rgb(21, 232, 99);
</string>
   </property>
   <property name="text">
    <string>Generate Comparison</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cmbTransport</sender>
   <signal>activated(QString)</signal>
   <receiver>mainEditorForm</receiver>
   <slot>transportSelect()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>89</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>455</x>
     <y>395</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmbDistance</sender>
   <signal>activated(QString)</signal>
   <receiver>mainEditorForm</receiver>
   <slot>distanceSelect()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>89</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel">
     <x>455</x>
     <y>395</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmbSpeed</sender>
   <signal>activated(QString)</signal>
   <receiver>mainEditorForm</receiver>
   <slot>speedSelect()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>89</x>
     <y>221</y>
    </hint>
    <hint type="destinationlabel">
     <x>455</x>
     <y>395</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chbPrice</sender>
   <signal>clicked()</signal>
   <receiver>mainEditorForm</receiver>
   <slot>priceCheck()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>92</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel">
     <x>455</x>
     <y>395</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chbEnergy</sender>
   <signal>clicked()</signal>
   <receiver>mainEditorForm</receiver>
   <slot>EnergyConsumptionCheck()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>155</x>
     <y>320</y>
    </hint>
    <hint type="destinationlabel">
     <x>455</x>
     <y>395</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chbConversions</sender>
   <signal>clicked()</signal>
   <receiver>mainEditorForm</receiver>
   <slot>EnergyConverCheck()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>155</x>
     <y>350</y>
    </hint>
    <hint type="destinationlabel">
     <x>455</x>
     <y>395</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnGen</sender>
   <signal>clicked()</signal>
   <receiver>mainEditorForm</receiver>
   <slot>generateCompare()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>445</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>455</x>
     <y>395</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>transportSelect()</slot>
  <slot>speedSelect()</slot>
  <slot>distanceSelect()</slot>
  <slot>priceCheck()</slot>
  <slot>EnergyConsumptionCheck()</slot>
  <slot>EnergyConverCheck()</slot>
  <slot>generateCompare()</slot>
 </slots>
</ui>
